querydsl-codegen/src/test/java/com/querydsl/codegen/GenericExporterTest.java

```diff
@@ -57,7 +57,7 @@ public void Export_With_Keywords() throws IOException {
     }
 
     @Test
-    public void Export_With_Stopclass() {
+    public void export_with_stopclass() {
```

Comment:
With preserve casing I'd say that `uncapSimpleName_escaped` is perfect, but this one would be `export_with_stopClass`.

What's your take on this? @johnktims @robertandrewbain